bin/src/commands.rs

```diff
@@ -1,3 +1,9 @@
+use ethui_db::utils::{fetch_etherscan_abi, fetch_etherscan_contract_name};
+use ethui_db::Db;
+use ethui_networks::commands::networks_is_dev;
+use ethui_sync::utils::provider;
```

Comment:
the commands (such as networks_is_dev) are meant to be like API endpoints, not public functions to call from other modules

and the sync utils are meant to stay private to that module. it's a bit arbitrary to just publicize that and use it here

what should happen here is to just depend on ethui_networks, fetch the network from there, and instantiate a provider (so, very similar to that sync::utils::provider is already doing)